core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts

```diff
@@ -204,13 +246,18 @@ describe('DotContainerPropertiesComponent', () => {
 
         it('should focus on title field', async () => {
             const inplace = de.query(By.css('[data-testId="inplace"]'));
+            inplace.componentInstance.activate();
+            fixture.detectChanges();
```

Comment:
Why? looks like you are manually activating the focus here and it should be auto.

Reply:
it is not activating the focus it shows the field of the title. In edit mode, you only see the title text, not the title field.